modules/ROOT/pages/keycloak/coding.adoc

```diff
@@ -90,4 +139,57 @@ try! AgsAuth.instance.configure(authConfig: authenticationConfig)
 .Cordova
 ****
 
-****
\ No newline at end of file
+Import and initialize Auth:
+
+[source,javascript]
+----
+const authService = new Auth();
+const initOptions = { onLoad: "login-required" };
+
+authService.init(initOptions)
+    .then(() => {
+        // successful init & authentication
+    })
+    .catch((err) => {
+        // initialization error
+    });
+----
+
+You can pass `login-required` or `check-sso` to the init function. `login-required` will authenticate the client if the user is logged in to Keycloak or display the login page if not. `check-sso` will only authenticate the client if the user is already logged in. If the user is not logged in the browser will be redirected back to the application and remain unauthenticated. By default, the `check-sso` option is used.
+
+NOTE: Initialization will also perform authentication
+
+****
+
+[role="secondary"]
+.Xamarin
+****
+*Initializing Auth in an Android app (MainActivity.cs)*
```

Comment:
We could reword a bit:
_For initializing Auth in Android, add the following code to your `MainActivity.cs`._
_For initializing Auth in iOS, add the following code to your `ApDelegate.cs`._